src/main/resources/templates/warehouse.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org" lang="da-dk">
+
+<head th:fragment="head(title)">
+    <title th:text="${title}">WarehousePage</title>
+    <meta charset="UTF-8">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../public/css/styles.css" th:href="@{/css/styles.css}" rel="stylesheet"/>
+    <link href="../public/css/warehouse.css" th:href="@{/css/warehouse.css}" rel="stylesheet"/>
+
+</head>
+
+<body>
+<div class="main-container">
+    <ul th:replace="~{fragments :: header('Login', ~{::li.nav-bar-button})}">
+        <li class="nav-bar-button">
+            <a href="/" th:href="@{/}">Tilbage</a>
+            <a class="nav-bar-button" href="/registerNewMaterial" th:href="@{/registerNewMaterial}">Registrer ny vare</a>
+        </li>
```

Comment:
There are two buttons in a single li tag. Each button must be in its own li tag and the a tag does not need the `nav-bar-button` class